python/tvm/relay/transform.py

```diff
@@ -657,6 +657,20 @@ def un_cps(func):
     return _transform.un_cps(func)
 
 
+def extract_fused_functions():
+    """Turn IRModule into IRModule of only fused primitive functions.
```

Comment:
Should we move it to relay.analysis instead since this is not an equivalent transformation?